module_utils/nxos/argspec/facts/facts.py

```diff
@@ -0,0 +1,7 @@
+from ansible.module_utils.network.argspec.base import ArgspecBase
+
+class FactsArgs(ArgspecBase):
+
+    argument_spec = {
+        'gather_subset': dict(default=['all'], type='list')
```

Comment:
Can we add individual supported keys, so that it is clear to the user which all option values it can support?

Reply:
agreed.